examples/CRISP/circuits/src/main.nr

```diff
@@ -16,8 +17,34 @@ use merkle_tree::get_merkle_root;
 mod utils;
 use utils::{check_coefficient_values_with_balance, check_coefficient_zero};
 
+/// Main circuit function for CRISP vote verification.
+///
+/// This circuit implements the core verification logic for the CRISP (Coercion-Resistant Impartial
+/// Selection Protocol) voting system. It verifies that votes are properly encrypted using BFV
+/// (Brakerski-Fan-Vercauteren) fully homomorphic encryption and handles two distinct use cases:
+/// 1. **Actual Voting**: An eligible voter casts a vote
+/// 2. **Mask Vote**: Anyone submits a zero vote to mask slot activity
+///
+/// Votes are stored in an on-chain key-value mapping, where the key is the slot address (Ethereum address)
```

Comment:
this is not true anymore, we store the vote in the merkle tree. In the mapping we store address => index in the tree